### PR TITLE
Fix mock module handling

### PR DESCRIPTION
**Pull request by dngroth**

### CODE REVIEW

File: Test.Utility/MockModule.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            foreach (var registration in _mockRegistrations)
+            {
+                registration(builder);
+            }
+        }
+    }
+}
```
Comment (bash) [inline Test.Utility/MockModule.cs]: Missing final newline

File: Test.Utility/MockModuleBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+            return this;
+        }
+
+        public IModule Build()
+        {
+            return new MockModule(_mockRegistrations);
+        }
+    }
+}
```
Comment (bash) [inline Test.Utility/MockModuleBuilder.cs]: Missing final newline

File: Test.Utility/MockModule.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    public class MockModule : Module
+    {
+        public delegate void Register(ContainerBuilder builder);
+
+        public MockModule(List<Register> mockRegistrations)
+        {
+            _mockRegistrations = mockRegistrations;
+        }
+
+        private readonly List<Register> _mockRegistrations;
```
Comment (bash) [inline Test.Utility/MockModule.cs]: Shouldn't fields come before the constructor(s)?

File: Test.Utility/MockModule.cs (hunk 1 above)
Comment (bash) [reply Test.Utility/MockModule.cs]: I see that this has been addressed in 35ae7c7d06bb738e4a059941c56416b5df15c9aa

File: Test.Utility/MockModuleBuilder.cs (hunk 2 above)
Comment (bash) [reply Test.Utility/MockModuleBuilder.cs]: I see that this has been addressed in 35ae7c7d06bb738e4a059941c56416b5df15c9aa

File: Test.Utility/MockModule.cs (hunk 3 above)
Comment (dngroth) [reply Test.Utility/MockModule.cs]: Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/4/commits/935655401a6aa3e6a2c119a85eb99b2602eb95be

File: changelog.md [hunk 4]
```diff
@@ -2,3 +2,8 @@
 
 ## 0.1.0
 - Initial release
+
+## Unreleased
+- Added MockModule
```
Comment (bash) [inline changelog.md]: Put class names between backticks

Comment (dngroth) [reply changelog.md]: Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/4/commits/ce756b192e37b7c6e076754d55ea7b2f0e72124b

**Pull request merged**